pkg/trace/traceutil/trace.go

```diff
@@ -109,24 +109,32 @@ func ChildrenMap(t pb.Trace) map[uint64][]*pb.Span {
 //
 // A span is considered top-level if:
 // - it's a root span
-// - its parent is unknown (other part of the code, distributed trace)
-// - its parent belongs to another service (in that case it's a "local root"
+// - OR its parent is unknown (other part of the code, distributed trace)
+// - OR its parent belongs to another service (in that case it's a "local root"
 //   being the highest ancestor of other spans belonging to this service and
 //   attached to it).
-func ComputeTopLevel(t pb.Trace) {
-	// build a lookup map
-	spanIDToIdx := make(map[uint64]int, len(t))
-	for i, span := range t {
-		spanIDToIdx[span.SpanID] = i
+func ComputeTopLevel(trace pb.Trace) {
+	spanIdToIndex := make(map[uint64]int, len(trace))
```

Comment:
This is fine, but please keep abbreviations all caps
```suggestion
	spanIDToIndex := make(map[uint64]int, len(trace))
```

Reply:
fixed :)